To: vim_dev@googlegroups.com
Subject: Patch 7.3.412
Fcc: outbox
From: Bram Moolenaar <Bram@moolenaar.net>
Mime-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
------------

Patch 7.3.412
Problem:    Storing a float in a session file has an additional '&'.
Solution:   Remove the '&'. (Yasuhiro Matsumoto)
Files:	    src/eval.c


*** ../vim-7.3.411/src/eval.c	2012-01-20 20:44:38.000000000 +0100
--- src/eval.c	2012-01-26 11:39:39.000000000 +0100
***************
*** 22929,22935 ****
  		    f = -f;
  		    sign = '-';
  		}
! 		if ((fprintf(fd, "let %s = %c&%f",
  					       this_var->di_key, sign, f) < 0)
  			|| put_eol(fd) == FAIL)
  		    return FAIL;
--- 22929,22935 ----
  		    f = -f;
  		    sign = '-';
  		}
! 		if ((fprintf(fd, "let %s = %c%f",
  					       this_var->di_key, sign, f) < 0)
  			|| put_eol(fd) == FAIL)
  		    return FAIL;
*** ../vim-7.3.411/src/version.c	2012-01-26 11:43:04.000000000 +0100
--- src/version.c	2012-01-26 11:44:56.000000000 +0100
***************
*** 716,717 ****
--- 716,719 ----
  {   /* Add new patch number below this line */
+ /**/
+     412,
  /**/

-- 
Imagine a world without hypothetical situations.

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///
